framework/src/bundle/CoreBundleContext.cpp

```diff
@@ -139,7 +139,15 @@ void CoreBundleContext::Init()
 //  {
 //    dataStorage = GetFileStorage(this, "data");
 //  }
-  dataStorage = GetFileStorage(this, "data");
+  try
```

Comment:
This is solution number two, which is also fine with me.

Reply:
You are right. I forgot to update the issue with this information. Will do that.